### ISSUE TITLE
Some extra dependecies neeed in some evironments

### ISSUE THREAD

**Issue by Pepe-Marquez** [njszym/XRD-AutoAnalyzer#12]
I do not know how reproducible this is, but in typical [data science jupyter environments ](https://hub.docker.com/r/jupyter/datascience-notebook) I get to run a couple of extra pip install lines:

```
!pip install git+https://github.com/njszym/XRD-AutoAnalyzer.git 
!pip install asteval 
!pip install numexpr --upgrade
```
Do you think it is needed to add `asteval` to the dependencies or pin a minimum version of `numexpr`?

**Comment by njszym** [comment]
Yes to `asteval` (I'll add this!) but I'm not sure about `numexpr` -- can you let me know where in the code you receive an error if that is not installed and up-to-date?

**Comment by Pepe-Marquez** [comment]
It is just a UserWarning that comes from the installed version of `pandas`:

```
UserWarning: Pandas requires version '2.8.4' or newer of 'numexpr' (version '2.8.3' currently installed).
  from pandas.core.computation.check import NUMEXPR_INSTALLED
```

**Comment by njszym** [comment]
Got it, added the version requirement, thanks for the info!
